var/spack/repos/builtin/packages/flex/package.py

```diff
@@ -31,10 +31,10 @@ class Flex(AutotoolsPackage):
     homepage = "https://github.com/westes/flex"
     url = "https://github.com/westes/flex/releases/download/v2.6.1/flex-2.6.1.tar.gz"
 
-    version('2.6.3', 'a5f65570cd9107ec8a8ec88f17b31bb1')
```

Comment:
Removing `2.6.3` is consistent with the previous removal of `2.6.2`.  Obviously, we cannot continue this forever.  Both should be added back, and then packages that don't work the new version should be set to use `@:2.6.1`.  Someone needs to grep for packages that use `flex`, and change/test them accordingly.

Also... versions should be in reverse numerical order.